thesisScalaProject/src/main/scala/be/emilesonneveld/MeasureProject.scala

```diff
@@ -109,7 +109,7 @@ object MeasureProject {
     colorGrade("//rect[@label=\"bg_LocPerNom\"]/@style", loc / nom, 4.71, 16.79, 10.75)
     colorGrade("//rect[@label=\"bg_CycloPerLoc\"]/@style", cc / loc, 0.1, 0.38, 0.24)
     colorGrade("//rect[@label=\"bg_CallsPerNom\"]/@style", calls / nom, 1.15, 2.42, 1.78)
-    colorGrade("//rect[@label=\"bg_FanoutPerCalls\"]/@style", fanout / calls, 1.02, 3.06, 2.04)
+    colorGrade("//rect[@label=\"bg_FanoutPerCalls\"]/@style", 0.404, 1.02, 3.06, 2.04)
```

Comment:
Is changing to a hardcoded value here intentionally? If so, would be nice to add a comment